src/ghdcbot/engine/pr_status.py

```
@@ -0,0 +1,541 @@
+    pr_lines: list[str] = []
+    for s in sorted_statuses:
+        health_emoji = _HEALTH_EMOJI.get(s.health_indicator, "❓")
+        detail_parts = []
+
+        # CI
+        ci_emoji = _CI_EMOJI.get(s.ci_status, "❓")
+        detail_parts.append(f"CI {ci_emoji}")
+
+        # Reviews
+        if s.review_state == "approved":
+            detail_parts.append(f"Reviews ✅")
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)